jkube-kit/enricher/generic/src/main/java/org/eclipse/jkube/enricher/generic/VolumePermissionEnricher.java

```diff
@@ -178,6 +187,33 @@ private String getMountPath(List<Container> containers, String name){
                 throw new IllegalArgumentException("No matching volume mount found for volume "+ name);
             }
 
+            private ResourceRequirements createNewResourceRequirements() {
+                if (hasNoResourcesConfigured())
+                    return null;
+                ResourceRequirementsBuilder resourceRequirementsBuilder = new ResourceRequirementsBuilder();
+                resourceRequirementsBuilder.withLimits(createResourcesMap(Config.CPU_LIMIT, Config.MEMORY_LIMIT));
+                resourceRequirementsBuilder.withRequests(createResourcesMap(Config.CPU_REQUEST, Config.MEMORY_REQUEST));
+                return resourceRequirementsBuilder.build();
```

Comment:
Any reason why we need a local `resourceRequirementsBuilder` variable?
Do you think it would look neater if we just returned the instantiated value?